### ISSUE TITLE
Deploy and pass CGP for cEUR oracle activation (staging)

### ISSUE THREAD

**Issue by nvtaveras** [celo-org/celo-monorepo#6167]
The goal of this governance proposal is to allow oracles to start reporting the CELO/EUR rate, while preventing activation of ExchangeCEUR. The proposal needs to contain these things:

- Freeze ExchangeCEUR
- Freeze StableTokenCEUR (TODO: determine if this is necessary or desirable)
- Add oracle addresses to the SortedOracle whitelist for cEUR
- Add methods to governance constitution.

**Comment by asaj** [comment]
This is blocked by the CR3 deployment to staging but there does not seem to be a ticket for that yet on the CAP team's backlog

**Comment by nambrot** [comment]
Is the blocking not the other way round?

**Comment by martinvol** [comment]
@nambrot without CR3 on staging this can't be deployed.

**Comment by martinvol** [comment]
Already done!
